File: 05-Problem-Variables/05-Problem-Variables.js
// Create a mutable variable and an immutable variable and print each in one log.  Add the immutable variable to the mutable variable. Again print both in a new log.

let mutable = 5;
const immutable = 6;

console.log(`mutable: ${mutable} immutable: ${immutable}`);

mutable+= immutable;

console.log(`mutable: ${mutable} immutable: ${immutable}`);

File: 05-Problem-Variables/05_Problem_Variables.py
# Create a mutable variable and an immutable variable and print each in one log.  Add the immutable variable to the mutable variable. Again print both in a new log.

mutable = 5
IMMUTABLE = 6

print(f"mutable: {mutable} immutable: {IMMUTABLE}")

mutable+= IMMUTABLE

print(f"mutable: {mutable} immutable: {IMMUTABLE}")
